src/Bridge/Doctrine/Orm/Extension/EagerLoadingExtension.php

```diff
@@ -67,17 +72,22 @@ public function __construct(PropertyNameCollectionFactoryInterface $propertyName
     /**
      * {@inheritdoc}
      */
-    public function applyToCollection(QueryBuilder $queryBuilder, QueryNameGeneratorInterface $queryNameGenerator, string $resourceClass, string $operationName = null)
+    public function applyToCollection(QueryBuilder $queryBuilder, QueryNameGeneratorInterface $queryNameGenerator, string $resourceClass = null, string $operationName = null, array $context = [])
     {
-        $options = null === $operationName ? [] : ['collection_operation_name' => $operationName];
+        if (null === $resourceClass) {
```

Comment:
What if `$resourceClass` is an empty string? I would prefer `if (empty($resourceClass))`
Same in all PR

Reply:
It would be a BC break (`''` is currently allowed).